src/main/kotlin/io/csbroker/apiserver/service/ProblemServiceImpl.kt

```diff
@@ -348,18 +349,23 @@ class ProblemServiceImpl(
         }.toList()
 
         // get score from content standards
-        val contentScores = findProblem.gradingStandards.filter {
+        val correctContentListDto = findProblem.gradingStandards.filter {
             it.type == GradingStandardType.CONTENT && it.id in gradeResultDto.correctContentIds
```

Comment:
```suggestion
            it.id in gradeResultDto.correctContentIds
```
어차피 gradeResultDto의 correctContentIds에 들은건 Content id들일텐데 이중으로 type filterring을 하는 것 보다는 애초에 저 correctContentIds에 content type이 아닌 id가 들어간건 설계 오류이니 exception 처리를 넣는게 좋지 않을까 라는 생각이 드네요!!